spectractor/extractor/spectrum.py

```diff
@@ -511,6 +511,44 @@ def load_chromatic_psf(self, input_file_name):
             self.my_logger.warning('\n\tSpectrogram file %s not found' % input_file_name)
 
 
+def open_header_for_adr(spectrum):
```

Comment:
Most of these keywords are already loaded in Image() (and then in Spectrum as it inherits from thr Image properties). I would prefer to modify the load of an image, create Image attributes, and pass to adr_calib explicitely the needed quantities, not a vector. I can do it if you prefer.

Reply:
I hope it's done

Reply:
ADR calculs on dispersion axis (not xaxis)